src/App.vue

```diff
@@ -268,6 +270,24 @@ export default class App extends Mixins(
     }
     await this.toggleConfirmNrModal(false)
   }
+
+  /** Whether the old webchat should be enabled. */
+  get enableOldWebchat (): boolean {
+    if (getFeatureFlag('enable-web-chat')) {
+      return !!getFeatureFlag('enable-web-chat')
+    } else {
+      return false // Failsafe value
+    }
```

Comment:
1. Please simplify this. Also below.

2. Add the failsafe values to https://github.com/bcgov/namerequest/blob/main/src/plugins/featureFlags.ts